src/views/pacbio/PacbioSampleIndex.vue

```diff
@@ -133,6 +137,9 @@ export default {
     printingStore() {
       return usePrintingStore()
     },
+    barcodes() {
+      return this.requests.map((request) => request.barcode).filter(Boolean)
```

Comment:
Same comment as ONT page - requests don't have barcodes.


EDIT: They do - They are added in the resource but only return Tube barcodes. This would require an update to also return plate barcodes. However ONT requests do not return barcodes.